### DISCUSSION TITLE
[Discussion] Versioning scheme

### DISCUSSION THREAD

**Discussion by Tastyep** [LunarVim/LunarVim#1786] [Ideas]
We've already started to discuss this a bit on discord, but since there are so many messages I feel it would be better to start a thread here.

Two versioning schemes have been discussed:
- [Semver](https://semver.org/)
- [Zerover]( https://0ver.org/)

I'm used to working with semver, after reading zerover spec multiple times I still fail to understand how it works.

**Comment by rebuilt** [comment]
Converting to zerover makes things easier.  "Q: When will LunarVim 1.0 be release? ", "A: Never".   

See?  Simple.  

From the zerover website:  
"ZeroVer is satire, please do not use it. We sincerely hope no project release schedules were harmed as a result of this humble attempt at programmer humor."

**Comment by Tastyep** [comment]
> From the zerover website:
> "ZeroVer is satire, please do not use it. We sincerely hope no project release schedules were harmed as a result of this humble attempt at programmer humor."

That's reassuring, while speaking of versioning we could also speak about the next release, does it really need to be 1.0 ?

**Comment by kylo252** [comment]
I'm not really sure about v1.0, and I don't think we can even version anything when the plugins themselves aren't versioned. 

Once you start pinning core-plugins, telescope and other, then you have to keep track to make sure you're always up-to-date with whatever critical-bugs they fixed in them. 

I believe that it will lead to more effort than necessary, when people actually want the _latest and greatest_.

**Comment by chaesngmin** [comment]
Yea maybe we should not version or pin plugins at all except for few cases.
And just update master branch regularly like every 2 weeks or whenever a bugfix needed.
